src/solvers/dgsem_tree/dg_3d_parabolic.jl

```diff
@@ -171,10 +172,10 @@ end
 
 # This is the version used when calculating the divergence of the viscous fluxes
 # We pass the `surface_integral` argument solely for dispatch
-function prolong2interfaces!(cache_parabolic, flux_viscous,
+function prolong2interfaces!(cache_parabolic, flux_viscous::Vector{Array{uEltype, 5}},
```

Comment:
See above on `flux_viscous`

Reply:
At this place not, this is some left-over.